Braintree.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```

Comment:
We dropped 32bit device support a long time ago (as did Apple). Since we don't need a check for that anymore, do we need this file?

Reply:
It seems like we still need it since it was generated automagically. We have one in [PPCP as well](https://github.com/paypal/iOS-SDK/blob/2b39110e75b5b8c4c79f16b76a6e5e010230f8ec/PayPal.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist#L5) so I am guessing if we don't include it it'll be generated and need to be ignored with each commit.

Reply:
Gotcha - thanks for checking!